Sources/App/WebView/WebViewController.swift

```diff
@@ -833,6 +833,18 @@ final class WebViewController: UIViewController, WKNavigationDelegate, WKUIDeleg
             }
         )
     }
+
+    public func openActionAutomationEditor(actionId: String) {
```

Comment:
if this requires a certain core version, you should gate the whole experience on that version